lib/purescript-cst/src/Language/PureScript/CST/Types.hs

```diff
@@ -153,8 +153,8 @@ data Type a
   deriving (Show, Eq, Ord, Functor, Foldable, Traversable, Generic)
 
 data TypeVarBinding a
-  = TypeVarKinded (Wrapped (Labeled (Name Ident) (Type a)))
-  | TypeVarName (Name Ident)
+  = TypeVarKinded (Maybe SourceToken) (Wrapped (Labeled (Name Ident) (Type a)))
```

Comment:
I don't know if it's clear, but in all other cases we preserve left-to-right order of tokens in the CST, which would imply:
```haskell
TypeVarKinded (Wrapped (Labeled (Maybe SourceToken, Name Ident) (Type a))
```